index.html

```diff
@@ -42,6 +42,14 @@ <h1>OnGrid FullStack Bootcamp</h1>
      <td>
       Varlamov
      </td>
+    </tr>
+       <tr>
+     <td>
+      Ilya
+     </td>
+     <td>
+      Kuzmenko
+     </td>
     </tr>
    </tbody>
```

Comment:
new `<tr></tr>` pair should be placed after last closing `</tr>` before `</tbody>`